tpu/templates/tools/parser_add.py

```diff
@@ -0,0 +1,67 @@
+import glob
+from redbaron import RedBaron
+
+
+help_dict = {
+    '--model-dir': 'Location to write checkpoints and summaries to.  Must be a GCS URI when using Cloud TPU.',
+    '--max-steps': 'The total number of steps to train the model.',
+    '--use-tpu': 'Whether to use TPU.',
+    '--tpu': 'The name or GRPC URL of the TPU node.  Leave it as `None` when training on CMLE.',
+    '--train-batch-size': 'The training batch size.  The training batch is divided evenly across the TPU cores.',
+    '--save-checkpoints-steps': 'The number of training steps before saving each checkpoint.',
+    '--sequence-length': 'The sequence length for an LSTM model.',
+    '--gr-weight': 'The weight used in the gradient reversal layer.',
+    '--lambda': 'The trade-off between label_prediction_loss and domain_classification_loss.'
+}
+
+filenames = glob.glob('../**/trainer*.py')
+
+for filename in filenames:
+    with open(filename) as f:
+        red = RedBaron(f.read())
+
+    # the `if __name__ == '__main__':` block
+    ifelseblock = red[-1]
+    nodes = ifelseblock.value[0]
+
+    for node in nodes:
+        # looking for those parser.add_argument calls
+        if node.type != 'atomtrailers':
+            continue
+
+        if node.value[0].name.value != 'parser' or node.value[1].name.value != 'add_argument':
```

Comment:
Does node.value always have the same number of elements? (3) Not sure if there is a possible case which node.value[X] is inaccessible and may cause exception.

Reply:
Added a comment pointing to the documentation on the node structure.